language/Portuguese_Brazil/strings.po

```diff
@@ -787,7 +787,7 @@ msgstr "Manter a Proporção do Aspecto"
 
 #: src/frontend/mame/ui/videoopt.cpp:205
 msgid "Cannot change options while recording!"
-msgstr "Não é possível alterar as opções durante a gravação!"
+msgstr "Não pode mudar as opções enquanto grava!"
```

Comment:
I also think this is worse than the previous text. Please don't make this change.

There's a lot of other changes, but I will stop my review here, because I do not consider this to be a good quality contribution to the Brazilian Portuguese translation. You should put additional effort to improve it and then request a new review once you think it is ready.